modules/core/src/main/java/org/apache/ignite/spi/metric/jmx/ReadOnlyDynamicMBean.java

```diff
@@ -57,16 +58,19 @@ else if ("invoke".equals(actionName))
 
     /** {@inheritDoc} */
     @Override public AttributeList getAttributes(String[] attributes) {
-        AttributeList list = new AttributeList();
-
+        AttributeList attributeList = new AttributeList();
```

Comment:
Please change `attributeList` to `attrList`
https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules